iosApp/Picover.xcodeproj/project.pbxproj

```diff
@@ -423,6 +454,8 @@
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"picover/Preview Content\"";
 				ENABLE_PREVIEWS = YES;
+				ENABLE_USER_SCRIPT_SANDBOXING = NO;
+				FRAMEWORK_SEARCH_PATHS = "$(SRCROOT)/../shared/build/xcode-frameworks/$(CONFIGURATION)/$(SDK_NAME)";
```

Comment:
`$CONFIGURATIONS` is Debug or Release

Example:
<img width="312" alt="image" src="https://github.com/Android-Guild/Picover-KMP/assets/12697488/3974b990-c1d2-4b74-bd10-1919a9b2756d">